src/actions/guides/database/models.cr

```diff
@@ -7,70 +8,160 @@ class Guides::Database::Models < GuideAction
 
   def markdown
     <<-MD
-    ## Setting up a model
+    ## Introduction
+
+    A Model is an object used to map a corresponding database table to a class. These objects model real-world objects to give you a better understanding on how they should interact within your application.
+
+    Models in Lucky allow you to define methods associated with each column in the table. These methods return the value set in that column.
```

Comment:
I think another paragraph saying `Models also generate classes you can use to save new records, and query existing ones` or something like that